share/management/commands/loadsources.py

```diff
@@ -132,7 +133,11 @@ def schedule_harvest_task(self, label, disabled):
     def process_defaults(self, model, defaults):
         ret = {}
         for k, v in defaults.items():
-            field = model._meta.get_field(k)
+            try:
```

Comment:
This is to handle the `canonical` field set in the source config that doesn't actually do anything?